pkg/validation/limits.go

```diff
@@ -294,6 +295,11 @@ func (o *Overrides) MaxQuerySeries(userID string) int {
 	return o.getOverridesForUser(userID).MaxQuerySeries
 }
 
+// MaxQueriersPerUser returns the maximum number of queriers that can handle requests for this user.
+func (o *Overrides) MaxQueriersPerUser(userID string) int {
```

Comment:
Can you add the doc of this in our docs and also include defaults like other in RegisterFlags please ?